pkg/queue/cluster_queue_strict_fifo.go

```diff
@@ -35,18 +35,18 @@ const StrictFIFO = kueue.StrictFIFO
 func newClusterQueueStrictFIFO(cq *kueue.ClusterQueue) (ClusterQueue, error) {
 	cqImpl := &ClusterQueueImpl{
 		heap: heapImpl{
-			less:  byCreationTime,
+			less:  byPriority,
 			items: make(map[string]*heapItem),
 		},
 	}
 	cqImpl.Update(cq)
 	return cqImpl, nil
 }
 
-// byCreationTime is the function used by the clusterQueue heap algorithm to sort
+// byPriority is the function used by the clusterQueue heap algorithm to sort
 // workloads. It sorts workloads based on their priority.
 // When priorities are equal, it uses workloads.creationTimestamp.
-func byCreationTime(a, b workload.Info) bool {
```

Comment:
`byPriorityThenCreationTime`?

Reply:
I can't find the previous discussion. Because the priority is necessary, so we delete the Priority in the function name.

Reply:
Let's close it then.